scss/_patterns_navigation.scss

```diff
@@ -763,6 +767,56 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  .p-navigation--reduced .p-navigation__dropdown.is-full-width,
+  .p-navigation--sliding .p-navigation__dropdown.is-full-width {
+    .p-navigation__dropdown-content {
+      @extend %vf-has-box-shadow;
+      @include vf-transition(transform, snap);
+      background-color: $colors--theme--background-default;
+      border-top: 1px solid $colors--theme--border-default;
+      display: none; // hide on small screens
+    }
+
+    // on desktop display full width meganav dropdowns
+    @media (min-width: $breakpoint-navigation-threshold) {
+      background-color: $color-transparent;
+      box-shadow: none;
+      left: 0;
+      overflow: hidden;
+      position: absolute;
+      right: 0;
+
+      // hide mobile sliding menu on full width view
+      .p-navigation__dropdown-content-mobile {
+        display: none;
+      }
+
+      .p-navigation__dropdown-content {
+        display: block;
+        transform: translateY(0);
+      }
+    }
+
+    &.is-collapsed .p-navigation__dropdown-content {
+      transform: translateY(-101%);
```

Comment:
Just checking, is this `tranform: translateY(-101%)` here for animation purposes or something else? It's not immediately clear from the code so it may be worth adding a comment

Reply:
Yes, it's for animation. I'll add something.